lib/logstash/inputs/gelf.rb

```diff
@@ -29,9 +30,12 @@ class LogStash::Inputs::Gelf < LogStash::Inputs::Base
   # The IP address or hostname to listen on.
   config :host, :validate => :string, :default => "0.0.0.0"
 
-  # The port to listen on. Remember that ports less than 1024 (privileged
+  # The ports to listen on. Remember that ports less than 1024 (privileged
```

Comment:
can I ask you to place these changes in the `docs/index.asciidoc` too? Essentially:
1. add to the [config list index](https://github.com/logstash-plugins/logstash-input-gelf/blob/master/docs/index.asciidoc#gelf-input-configuration-options) the `port_tcp`, `port_udp`, `use_tcp` and `use_udp` settings
2. mention in [the existing port setting description](https://github.com/logstash-plugins/logstash-input-gelf/blob/master/docs/index.asciidoc#port) that per protocol port can be set (line 35 of this PR)
3. add the 4 new config setting sections for `port_tcp`/`port_udp` and `use_tcp`/`port_udp`

the first two are similar to:

```
[id="plugins-{type}s-{plugin}-port"]
===== `port` 

  * Value type is <<number,number>>
  * Default value is `12201`

The port to listen on. Remember that ports less than 1024 (privileged
ports) may require root to use.
```

the boolean settings will be similar to 
```
[id="plugins-{type}s-{plugin}-strip_leading_underscore"]
===== `strip_leading_underscore` 

  * Value type is <<boolean,boolean>>
  * Default value is `true`

Whether or not to remove the leading `\_` in GELF fields or leave them
in place. (Logstash < 1.2 did not remove them by default.). Note that
GELF version 1.1 format now requires all non-standard fields to be added
as an "additional" field, beginning with an underscore.

e.g. `\_foo` becomes `foo`
```

Reply:
Done